### DISCUSSION TITLE
RTSP Control and seekable streaming

### DISCUSSION THREAD

**Discussion by iolumbaram** [bluenviron/mediamtx#325] [General]
**## Which version are you using?**

v0.14.1

**## Question**

How to create a seekable streaming and have controls using RTSP such that a playback can pause (this works), or seek to a different position, or enter fast forward or rewind mode.

**Description**

I have a RTSP streaming that is published to RTSP server at localhost:8554. I can pull the stream from rtsp://localhost:8554/mystream on VLC player. I have tested VLC player to pull the stream, it is showing but only is able to pause and not not able to perform other controls. 

For your helps, thank you!

**Comment by napanto** [comment]
@aler9 any update on this?

**Reply by aler9** [reply]
Stream playback is provided through the Playback server, documented here:
https://github.com/bluenviron/mediamtx?tab=readme-ov-file#playback-recorded-streams

Playback through RTSP won't be developed.
